<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ada</class>
 <widget class="QMainWindow" name="ada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Analog Data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>691</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Channel 1:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ch1label">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Channel 2:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ch2label">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Channel 3:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ch3label">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage1" native="true">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="gridCheckBox">
         <property name="text">
          <string>Grid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoscaleCheckBox">
         <property name="text">
          <string>Autoscale</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="yminSpinBox">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>V</string>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>â€“</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ymaxSpinBox">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>V</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;V&lt;/span&gt;&lt;span style=&quot; font-size:10pt; vertical-align:sub;&quot;&gt;ref&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="vrefDoubleSpinBox">
         <property name="suffix">
          <string>V</string>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="viewsizeSpinBox">
         <property name="suffix">
          <string>s</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Scroll</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="scrollSpinBox">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2" native="true">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portLineEdit">
         <property name="text">
          <string>/dev/ttyUSB0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="speedLabel">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="speedLineEdit">
         <property name="text">
          <string>9600</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="intervalLabel">
         <property name="text">
          <string>Interval:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intervalSpinBox">
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="connectCheckBox">
         <property name="text">
          <string>Autoconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="connectLabel">
         <property name="text">
          <string>not connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>gridCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ada</receiver>
   <slot>gridEnable(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoscaleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ada</receiver>
   <slot>setAutoscale(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yminSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ada</receiver>
   <slot>setYmin(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ymaxSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ada</receiver>
   <slot>setYmax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vrefDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ada</receiver>
   <slot>setVref(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewsizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ada</receiver>
   <slot>setView(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrollSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ada</receiver>
   <slot>setScroll(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoscaleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>yminSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoscaleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ymaxSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gridEnable(bool)</slot>
  <slot>setView(int)</slot>
  <slot>setScroll(int)</slot>
  <slot>setYmin(int)</slot>
  <slot>setYmax(int)</slot>
  <slot>setAutoscale(bool)</slot>
  <slot>setVref(double)</slot>
 </slots>
</ui>
